Tasks/AzureCloudPowerShellDeployment/Publish-AzureCloudDeployment.ps1

```diff
@@ -233,7 +233,7 @@ if (!$azureService)
     $azureService = Invoke-Expression -Command $azureService
 }
 
-$diagnosticExtensions = Get-DiagnosticsExtensions $StorageAccount $CsCfg
+$diagnosticExtensions = Get-DiagnosticsExtensions $StorageAccount $serviceConfigFile
```

Comment:
What if find-files doesn't resolve. I'm guessing we would error out, but just want to confirm that your change wouldn't introduce a worse error message.